integration-tests/load/automationv2_1/automationv2_1_test.go

```diff
@@ -292,21 +281,16 @@ Load Config:
 	numberOfUpkeeps := *loadedTestConfig.Automation.General.NumberOfNodes
 
 	for i := 0; i < numberOfUpkeeps+1; i++ { // +1 for the OCR boot node
-		var nodeTOML string
-		if i == 1 || i == 3 {
-			nodeTOML = fmt.Sprintf("%s\n\n[Log]\nLevel = \"%s\"", baseTOML, *loadedTestConfig.Automation.General.ChainlinkNodeLogLevel)
-		} else {
-			nodeTOML = fmt.Sprintf("%s\n\n[Log]\nLevel = \"info\"", baseTOML)
-		}
-		nodeTOML = networks.AddNetworksConfig(nodeTOML, loadedTestConfig.Pyroscope, testNetwork)
```

Comment:
Would a test with `pyroscope` enabled work if with this?
Running a test https://github.com/smartcontractkit/chainlink/actions/runs/9644338945/job/26596224343 to confirm

Reply:
confirmed it works both when enabled and disabled.